pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[project]
+name = "rapyuta-io-sdk-v2"
+version = "0.1.0"
```

Comment:
@pallabpain Can we start this with version 2.0.0?

Reply:
We can start it with version 1.0.0 when we roll out the first release.

The current SDK is named `rapyuta_io`. We can probably publish it as `rapyuta_io_v2`.

**Importing the module**
```python
import rapyuta_io_v2
```
**requirements.txt**
```
rapyuta-io-v2==1.0.0
```